spec/index.html

```diff
@@ -2036,7 +2036,7 @@ <h3>Algorithm</h3>
                   recursively processed by this algorithm.</li>
                 <li id="hndq-5-4-4">For each <var>related</var> in <var>p</var>:
                   <ol>
-                    <li id="hndq-5-4-4-1">If a canonical identifier has been issued for
+                    <li id="hndq-5-4-4-1">If an identifier has been issued for
                       <var>related</var> by <var>issuer</var>, append the string <code>_:</code>, followed by 
```

Comment:
```suggestion
                    <li id="hndq-5-4-4-1">If a canonical identifier has been issued for
                      <var>related</var> by <a>canonical issuer</a>, append the string <code>_:</code>, followed by 
```

Reply:
Ah! So there was indeed a discrepancy, but just not where I thought it was...